### PR TITLE
Unreal performance update

### PR DESCRIPTION
**Pull request by hferrone**

### CODE REVIEW

File: mixed-reality-docs/performance-recommendations-for-unreal.md [hunk 1]
```diff
@@ -23,15 +23,41 @@ You can find each of the following settings in **Edit > Project Settings**.
...
 
-2. Disabling occlusion culling:
+2. Using Forward Rendering: 
+    <!-- TODO: Where is this setting? -->
+    * Much better for MR than the default Deferred rendering pipeline, primarily because of the number of features that can then be individually turned off. You can find more details in the [Unreal documentation](https://docs.unrealengine.com/Platforms/VR/DevelopVR/VRPerformance/index.html).
+
+![Mobile target setting](images/unreal/performance-recommendations-img-04.png)
+
+<!-- TODO: Where is this setting? -->
+3. Disable vertex fogging is your application doesn't need it:
```
Comment (sw5813) [inline mixed-reality-docs/performance-recommendations-for-unreal.md:33]: Typo- "is" should be "if"

File: mixed-reality-docs/performance-recommendations-for-unreal.md [hunk 2]
```diff
@@ -23,15 +23,41 @@ You can find each of the following settings in **Edit > Project Settings**.
...
 
-4. Setting the **Maximum number of CSM cascades to render** to **1** and **Max Movable Spotlights / Point Lights** to **0**. 
+<!-- TODO: Where is this setting? -->
+7. Reducing the number of shadow maps:
+    * Set the **Maximum number of CSM cascades to render** to **1**, unless there is a visible quality loss, and **Max Movable Spotlights / Point Lights** to **0**:
+
+![VR rendering settings](images/unreal/performance-recommendations-img-07.png)
+
+<!-- TODO: Where is this setting? -->
+8. Make sure the **Start in VR** setting is checked when setting up a new scene:
```
Comment (sw5813) [inline mixed-reality-docs/performance-recommendations-for-unreal.md:61]: This setting should be removed from this doc, as it's not performance-related. I'll add this step to the tutorial and deploy to device docs.

File: mixed-reality-docs/performance-recommendations-for-unreal.md [hunk 3]
```diff
@@ -23,15 +23,41 @@ You can find each of the following settings in **Edit > Project Settings**.
 
 ![Mobile target setting](images/unreal/performance-recommendations-img-01.png)
 
-2. Disabling occlusion culling:
+2. Using Forward Rendering: 
+    <!-- TODO: Where is this setting? -->
```
Comment (sw5813) [inline mixed-reality-docs/performance-recommendations-for-unreal.md:27]: All of the settings are under Project Settings > Rendering

File: mixed-reality-docs/performance-recommendations-for-unreal.md (hunk 2 above)
Comment (hferrone) [reply mixed-reality-docs/performance-recommendations-for-unreal.md:61]: Perfect!

**Pull request closed without merging**